### PR TITLE
Docsify site

### PR DESCRIPTION
**Pull request by gvolpe**
Here are some screenshots (they look dark because I use the Dark Theme browser extension but most people will see the light colors version).

### CODE REVIEW

**Comment by mpilquist** [comment]
Awesome! Is there anything special to do to make sure when I merge `main` to `develop` branch, the site doesn't publish on each update to develop branch?

**Comment by SystemFw** [comment]
> Awesome! Is there anything special to do to make sure when I merge main to develop branch, the site doesn't publish on each update to develop branch?

Does the build publish hash releases on each merge to main now?

**Comment by mpilquist** [comment]
The regular CI build does, on both main and develop branches.

**Comment by SystemFw** [comment]
> The regular CI build does, on both main and develop branches.

Ok, so for one we need to change the build so that the microsite isn't published on hash releases (imho it should reflect the latest named release). We also need to figure out which branch do want to publish it from (I assume `main`, for the time being). That will also need some tweaking to the `cond` in that release postamble definition

**Comment by mpilquist** [comment]
That might be hard with sbt-spiewak/sbt-github-actions, unless we add some special conditional logic to build.sbt. A named release is accomplished via pushing a tag so we'd need logic to say only publish site on a `x.y.z` release, not a `x.y.z-M*` or `x.y.z-RC*`

**Comment by SystemFw** [comment]
Look at https://github.com/typelevel/fs2/pull/2223/files?file-filters%5B%5D=.css&file-filters%5B%5D=.html&file-filters%5B%5D=.md&file-filters%5B%5D=.nix&file-filters%5B%5D=.png&file-filters%5B%5D=.sbt&file-filters%5B%5D=.svg#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R318-R322

we need to change that condition to

> to say only publish site on a x.y.z release, not a x.y.z-M* or x.y.z-RC*

which might need some fiddling yeah

File: build.sbt [hunk 1]
```diff
@@ -317,3 +286,47 @@ lazy val experimental = project
     osgiSettings
   )
   .dependsOn(coreJVM % "compile->compile;test->test")
+
+lazy val microsite = project
```
Comment (diesalbla) [inline build.sbt:290]: You may also want to remove the entry from `plugins.sbt` file.

Comment (gvolpe) [reply build.sbt:290]: Yeah good catch :+1: will do

**Review by diesalbla — APPROVED** [review]

**Pull request merged**